AOU_RW_MODULE_WALKTHROUGH.md

```diff
@@ -0,0 +1,276 @@
+# AoU Researcher Workbench Module  Walkthrough
+## 0. Module Structure
+The state associated with the current deployment  consists of
+one `root` module for each environment, in separate directories
+
+In  order to deploy a full (or partial) environment we need to declare what modules are used and to supply
+values to all unbound declared variables. The environment module is  unioned with the modules in the
+`source` statement.
+
+The overall source structure looks like the following. Note that 
+Terraform will collect all `.tf` files in a referenced directory,
+so the calling module will need to specify values for the chilid
+modules' `variable` blocks that don't have defaults.
+
+```text
+/repos/workbench/ops/terraform/
+├── AOU_RW_MODULE_WALKTHROUGH.md
+├── TERRAFORM-QUICKSTART.md
+├── environments
```

Comment:
Is this directory supposed to exist? Or was this from a prior version?

Reply:
It's old, sorry. Those have moved to the `workbench-devops` repo under `terraform-live`.